plugins/generator-1.19.4/forge-1.19.4/generator.yaml

```diff
@@ -67,6 +67,19 @@ base_templates:
   - template: elementinits/javamodels.java.ftl
     name: "@SRCROOT/@BASEPACKAGEPATH/init/@JavaModNameModels.java"
     condition: hasJavaModels()
+  # Templates for custom structure feature

```

Comment:
Given the generation condition is `${w.hasElementsOfType('feature')}`, couldn't those files be part of feature.definition.yaml in `global_templates:` section?

Reply:
I am also wondering if maybe it would be less invasive to make one structure feature for each feature ME that has a structure feature in XML.

Or maybe only generate if any feature has structure feature in XML, as otherwise most MCreator mods will contain custom structure classes.